a/azure-sdk-for-python/azure-mgmt-authorization_0.51.1_ubi_8.txt

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : azure-mgmt-authorization
+# Version       : 0.51.1
+# Source repo   : https://github.com/Azure/azure-sdk-for-python
+# Tested on	: UBI 8.5
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Vaibhav Bhadade <vaibhav.bhadade@ibm.com> 
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+PACKAGE_NAME=azure-mgmt-authorization
+PACKAGE_VERSION=${1:-azure-mgmt-authorization_0.51.1}
+PACKAGE_URL=${2:-https://github.com/Azure/azure-sdk-for-python}
+PACKAGE_FOLDER=azure-sdk-for-python
+
+yum install -y git  python3 python3-devel make gcc-c++ rust-toolset openssl openssl-devel libffi libffi-devel 

```

Comment:
Please check the dependencies, Do we need all of them (like `gcc-c++`) ?
As per repo description, below are the languages used :
![image](https://user-images.githubusercontent.com/69784217/151950875-85ccd05f-1074-48e2-be0b-3822a193decf.png)